modules/op-installing-pipelines-operator-in-web-console.adoc

```diff
@@ -22,7 +22,7 @@ The supported profiles are:
 
 * Lite: This installs only Tekton Pipelines.
 * Basic: This installs Tekton Pipelines, Tekton Triggers, and Tekton Chains.
-* All: This is the default profile used when the `TektonConfig` CR is installed. This profile installs all of the Tekton components, including Tekton Pipelines, Tekton Triggers, Tekton Chains, {pac}, and Tekton Addons. Tekton Addons includes the `ClusterTasks`, `ClusterTriggerBindings`, `ConsoleCLIDownload`, `ConsoleQuickStart`, and `ConsoleYAMLSample` resources.
+* All: This is the default profile used when the `TektonConfig` CR is installed. This profile installs all of the Tekton components, including Tekton Pipelines, Tekton Triggers, Tekton Chains, {pac}, and Tekton Addons. Tekton Addons includes the `ClusterTasks`, `ClusterTriggerBindings`, `ConsoleCLIDownload`, `ConsoleQuickStart`, and `ConsoleYAMLSample` resources, as well as the tasks available using the Cluster resolver from the `openshift-pipelines` namespace.
```

Comment:
> , as well as the tasks available using the Cluster resolver from the `openshift-pipelines` namespace.

how about this
`, as well as the tasks available from the `openshift-pipelines` namespace.`

as this will be the functionality of Tekton Addon 

cc @vdemeester

Reply:
Isn't Chains missing in the list?

Reply:
Chains is present in the list.

Reply:
I'm sorry, I didn't notice it